packages/twenty-front/src/modules/object-metadata/schemas/camelCaseStringSchema.ts

```diff
@@ -0,0 +1,8 @@
+import camelCase from 'lodash.camelcase';
+import { z } from 'zod';
+
+export const camelCaseStringSchema = z
```

Comment:
move this to src/utils/validation-schemas

Reply:
Done!